svix-cli/src/cmds/open.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::Context;
+use clap::{Args, Subcommand};
+
+#[derive(Args)]
+#[command(args_conflicts_with_subcommands = true)]
+#[command(flatten_help = true)]
+pub struct OpenArgs {
+    #[command(subcommand)]
+    pub command: OpenCommands,
+}
+
+const API_DOCS_URL: &str = "https://api.svix.com/docs";
+const DOCS_URL: &str = "https://docs.svix.com/";
+
+/// Quickly open Svix pages in your browser
+#[derive(Subcommand)]
+pub enum OpenCommands {
+    /// Open the Svix API documentation
```

Comment:
Maybe this would make it easier to tell the difference between "api" and "docs"?

```suggestion
    /// Open the Svix API reference
```